### DISCUSSION TITLE
console & csv column output for sigma

### DISCUSSION THREAD

**Discussion by infosecgrab** [WithSecureLabs/chainsaw#111] [Ideas]
Can we configure the column output ? I'd love to have the criticity level column for detection output

**Comment by alexkornitzer** [comment]
Currently only columns from the data sources can be customised. To see those of the rule you can run Chainsaw with the `--metadata` flag. This will only do that for the cli tabular output, if you would like it in the CSV output too then we can add that in, should be very simple to do.
